collect_app/build.gradle

```diff
@@ -62,8 +62,8 @@ if (secretsFile.exists()) {
 }
 
 android {
-    compileSdkVersion(25)
-    buildToolsVersion('26.0.1')
+    compileSdkVersion(27)
+    buildToolsVersion('26.0.2')
```

Comment:
We should make sure we match this with Docker image. I'll follow up with @jknightco about this.